engine/src/test/java/org/pentaho/di/www/CarteTest.java

```diff
@@ -110,24 +110,24 @@ public void test() throws Exception {
 
   @Test
   public void callStopCarteRestService() throws Exception {
```

Comment:
Ignore this testcase if it is not working. I will try to do another pull request once this is merged